elrond.md

```diff
@@ -936,6 +969,40 @@ TODO: Implement [reserved keys and read-only runtimes](https://github.com/Elrond
          </locals>
 ```
 
+### Crypto EI
```

Comment:
What does EI mean?

Reply:
no idea. it just appear in the their filename. I'll remove it